### PR TITLE
issues##3769 Update Documentation for Forbidden configuration prefixes

### PR DESCRIPTION
**Pull request by codeatri**
### Type of change

_Select the type of your PR_

- Documentation

### Description

Forbidden configuration prefixes missing from Cruise Control configuration documentation

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

**Comment by ppatierno** [comment]
@codeatri can you fix the DCO please?

**Review by ppatierno — CHANGES_REQUESTED** [review]
As suggested by @kyguy in #3769 I would avoid fixing this list every time we do a change.
I agree with having a link to the auto-generated https://strimzi.io/docs/operators/latest/using.html#type-CruiseControlSpec-reference is much better.

File: documentation/modules/cruise-control/ref-cruise-control-configuration.adoc [hunk 1]
```diff
@@ -14,25 +14,7 @@ The `config` property in `Kafka.spec.cruiseControl` contains configuration optio
...
-* `metric.reporter.sampler.bootstrap.servers`
-* `metric.reporter.topic`
-* `metric.reporter.topic.pattern`
-* `partition.metric.sample.store.topic`
-* `broker.metric.sample.store.topic`
-* `capacity.config.file`
-* `skip.sample.store.topic.rack.awareness.check`
-* `cruise.control.metrics.topic`
-* `sasl.`
+Specifically, you *cannot* modify configuration options with keys equal to or starting with one of the keys mentioned https://strimzi.io/docs/operators/latest/using.html#type-CruiseControlSpec-reference[here.]
```
Comment (scholzj) [inline documentation/modules/cruise-control/ref-cruise-control-configuration.adoc:17]: Could you please update it to relative link?

```suggestion
Specifically, you *cannot* modify configuration options with keys equal to or starting with one of the keys mentioned xref:type-CruiseControlSpec-reference[here].
```

**Review by kyguy — APPROVED** [review]
Looks good! Just need to address up Jakub's suggestion and we should be good to go. Thanks for the fix @codeatri!

**Review by ppatierno — APPROVED** [review]
LGTM apart from Jakub's suggestion. Thanks!

**Comment by codeatri** [comment]
Thanks for the review everyone.

Updated PR with Jakub's suggestion.

**Review by scholzj — APPROVED** [review]

**Comment by scholzj** [comment]
Thanks for the PR.

**Pull request merged**